src/Gjallarhorn/State.fs

```diff
@@ -82,3 +83,34 @@ type State<'TModel,'TMsg> (initialState : 'TModel, update : 'TMsg -> 'TModel ->
         member this.Dispose() = 
             (signal :?> System.IDisposable).Dispose()            
             GC.SuppressFinalize this
+
+/// A thread-safe wrapper for a reference value with change notification
+type AtomicMutable<'a when 'a : not struct>(value : 'a) as self =
```

Comment:
Is the struct requirement needed?  I thought that was the entire reason for the ref cell (so it'd work on structs)